qa/L0_parameters/test.sh

```diff
@@ -62,12 +67,48 @@ if [ $? -ne 0 ]; then
     cat $CLIENT_LOG
     echo -e "\n***\n*** Test Failed\n***"
     RET=1
+else
+    check_test_results $TEST_RESULT_FILE $EXPECTED_NUM_TESTS
+    if [ $? -ne 0 ]; then
+        cat $CLIENT_LOG
+        echo -e "\n***\n*** Test Result Verification Failed\n***"
+        RET=1
+    fi
 fi
 set -e
 
 kill $SERVER_PID
 wait $SERVER_PID
 
+export TEST_HEADER=1
```

Comment:
To avoid code duplication, can we loop through the values of TEST_HEADER 0 and 1. 
If TEST_HEADER is 1 then set the extra SERVER_ARGS.